### ISSUE TITLE
API upload media

### ISSUE THREAD

**Issue by mikbry** [Zoapp/backend#4]
add a /media end point
POST : to upload a picture
GET to get a media
DELETE : to delete

**Comment by willdurand** [comment]
Where do we store the files? Filesystem? Should it be configurable? Can we imagine changing the storage type at some point?

**Comment by mikbry** [comment]
yes filesystem, path configurable. I was thinking of something similar to Wordpress. And later yes use another storage like Amazon S3

**Comment by willdurand** [comment]
> I was thinking of something similar to Wordpress.

what do you mean?

**Comment by mikbry** [comment]
the storage file path
